workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/Readers/IReaderInternal.cs

```diff
@@ -6,11 +6,7 @@ namespace Improbable.Gdk.Core.MonoBehaviours
     ///     The interface that is used to signal a reader that a message has been received from the SpatialOS Dispatcher
     ///     about the component that the reader is connected to.
     /// </summary>
-    internal interface IReaderInternal
+    internal interface IReaderWriterInternal
```

Comment:
The filename should change as well, and the in-code documentation above should be updated

Reply:
good catch :)